### ISSUE TITLE
Prisma has a very slow startup time when my database has 100+ tables

### ISSUE THREAD

**Issue by meotimdihia** [prisma/prisma#10236]
### Bug description

I am using `ts-node-dev` to run Fastify in the development environment.
when I modified my application code => my application restarted, and it took 10s to complete because of  Prisma.

This problem is frustrating, it is affecting development speed. 😥
Do we have a way to solve this problem - something like separate Prisma with the application...

https://user-images.githubusercontent.com/300961/141484875-8f885337-583f-4a4c-98cc-7814d0aabdbb.mp4

I think it is my PC is not very high performance but even with high spec PC, I doubt it still takes 3-5s to start.

### How to reproduce

I tried to create a new simple project with the existing database with the code takes 4s to finish: 
```
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

async function main() {
     console.log(    await prisma.users.findMany())
}

main()
  .catch((e) => {
    throw e
  })
  .finally(async () => {
    await prisma.$disconnect()
  })
 ```
` ts-node index.ts`

### Prisma information

Nothing is special: 
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["interactiveTransactions"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

### Environment & setup

- OS: Windows 10
- Database: Postgresql
- Node.js version: 14
- SSD benchmark speed: read 500MB/s
- CPU benchmark score: 7000

### Prisma Version

```
$ C:\sources\test\node_modules\.bin\prisma -v
Environment variables loaded from .env
prisma                  : 3.4.2
@prisma/client          : 3.4.2
Current platform        : windows
Query Engine (Node-API) : libquery-engine 57771c0558568c7d08bd34c7248af5244ae16bd9 (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Migration Engine        : migration-engine-cli 57771c0558568c7d08bd34c7248af5244ae16bd9 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine    : introspection-core 57771c0558568c7d08bd34c7248af5244ae16bd9 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary           : prisma-fmt 57771c0558568c7d08bd34c7248af5244ae16bd9 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash    : 57771c0558568c7d08bd34c7248af5244ae16bd9
Studio                  : 0.438.0
```

**Comment by janpio** [comment]
Can you share your schema so we can try to reproduce this?

**Comment by meotimdihia** [comment]
@janpio How can I send it in private?

**Comment by meotimdihia** [comment]
@janpio I have sent for you via Twitter message 😅.

**Comment by janpio** [comment]
Unconventional, but got it :D 

Will set up a reproduction and see what is going on.

**Comment by meotimdihia** [comment]
Sorry, I have found out the problem by enabling debug log. This happened because Sequelize (that I used in past time) when I included a Prisma lib.
I cut the compile time by 1/2 when I removed it.

**Comment by janpio** [comment]
Thanks for the update. Indeed good that you could figure it out, because on our side this looked pretty fast and simple enough.

Can you elaborate and explain a bit exactly what happened? Was Sequelize opening a database connection each time maybe? We are generally very interested in making it as simple as possible for our users to move from Sequelize to Prisma, but also to use both at the same time.

**Comment by meotimdihia** [comment]
@janpio Sequelize just takes an additional 2s to load all models for me. I have found out other libs are causing this problem too, like takes 1s to load Amazon S3 libs. 

After all optimization, I reduced the Fastifly server load time from 12s -> 4s. It is good for me now.

It is my inexperience  in debug a nodejs app: `ts-node-dev -r tsconfig-paths/register  --transpile-only --debug src/web/index.ts`
